_data/sites.json

```
@@ -376,6 +376,18 @@
         ]
     },
 
+    {
+        "name": "AirMeet",
+        "url": "https://help.airmeet.com/support/solutions/articles/82000443799-how-to-delete-airmeet-community-account-",
+        "difficulty": "hard",
+        "notes": "Click the purple question mark button and then request account deletion via the Airmeet Chat Support. Alternatively, you can email support@airmeet.com.",
+        "notes_pt": "Clique no botão roxo com um ponto de interrogação no canto inferior direito da página e inicie um Airmeet Chat Support. Alternativamente, envie um e-mail para support@airmeet.com.",
```

```suggestion
        "notes_pt_br": "Clique no botão roxo com um ponto de interrogação no canto inferior direito da página e inicie um Airmeet Chat Support. Alternativamente, envie um e-mail para support@airmeet.com.",
```

We only support `pt_br` currently, but this description is good as is for it too, so I'm changing it.